frontend/src/components/input.tsx

```diff
@@ -36,23 +55,43 @@ export const Input = ({ sendMessage, waiting }: InputProps) => {
   return (
     <>
       <form onSubmit={onSend} className={styles.inputContainer}>
-        <input
-          className={styles.input}
-          onChange={onChange}
-          onSubmit={onSend}
+        <textarea
+          className={styles.textarea}
+          ref={textareaRef}
           placeholder="Send a Message..."
-          type="text"
           value={userInput}
+          onChange={onChange}
+          rows={1}
         />
-        <button
-          className={classNames(styles.sendButton, {
+        <div
+          className={classNames(styles.uploadButton_container, {
+            [styles.disabled]: sendDisabled,
+          })}
+        >
+          <button
```

Comment:
Potentially this style of button could also be made into its own component or as a configurable option of the main button component, either way, something for a future PR. We may have more kinds of this button in the future